indico/modules/events/tracks/controllers.py

```diff
@@ -63,3 +68,39 @@ def _process(self):
                                 tracks=[{'id': int(t.getId()), 'title': to_unicode(t.getTitle())}
                                         for t in self._conf.getTrackList()])
         return jsonify_form(form)
+
+
+class RHEditTrack(RHManageTracksBase):
+
+    def _checkParams(self, params):
+        RHConferenceModifBase._checkParams(self, params)
+        self.track = Track.find_one(id=request.view_args['id'])
+
+    def _process(self):
+        form = TrackForm(obj=FormDefaults(self.track))
+        if form.validate_on_submit():
+            update_track(self.track, form.data)
+            return jsonify_data(flash=False)
+        return jsonify_form(form)
+
+
+class RHSortTracks(RHManageTracksBase):
+
+    def _process(self):
+        sort_order = request.json['sort_order']
```

Comment:
just fyi, you could also use  `request.form.getlist('sort_order')` like we do in [many other](https://github.com/indico/indico/blob/master/indico/modules/events/contributions/controllers/management.py#L382-L388) sorting RHs